### PR TITLE
Krhubert/core v2

### PR DESCRIPTION
**Pull request by NicolasMahe**
I had to recreate this branch and PR because  #333 has been merged into #332 in a bad way (the modification on package core had been reverted).

### CODE REVIEW

File: api/core/execute.go [hunk 1]
```diff
@@ -1,32 +1,30 @@
...
 func (s *Server) ExecuteTask(ctx context.Context, request *ExecuteTaskRequest) (*ExecuteTaskReply, error) {
 	service, err := services.Get(request.ServiceID)
 	if err != nil {
 		return nil, err
 	}
+
 	var inputs map[string]interface{}
-	err = json.Unmarshal([]byte(request.InputData), &inputs)
-	if err != nil {
+	if err = json.Unmarshal([]byte(request.InputData), &inputs); err != nil {
```
Comment (ilgooz) [inline api/core/execute.go]: usually errs recreated when used within ifs like this `err :=`

Comment (NicolasMahe) [reply api/core/execute.go]: Done

**Comment by NicolasMahe** [comment]
@antho1404 can you review and merge please

**Comment by NicolasMahe** [comment]
@antho1404 please don't forget to close the branch after merge

**Pull request merged**